### ISSUE TITLE
Redirects: RAD FAQs

### ISSUE THREAD

**Issue by bmathesonFEC** [fecgov/fec-cms#3350]
### What we are after:
Identify and put in redirects to RAD FAQs. Best link for RAD to use in RFAIs.

RAD pages:
- [x] Identify a replacement link for RAD's RFAIs.
- [x] Identify pages that need to be redirected ([enter on spreadsheet](https://docs.google.com/spreadsheets/d/1Jn5nC3fFku10RTCP5OAW6RiffjNBE4VwBubITQpCNK8/edit#gid=1272122359)) and where to redirect to (proxy and do we need a www or Wagtail redirect too)
- [ ] After redirect is entered, put a testing ticket based on the release schedule
- [ ] Put in ticket for front end to take down the pages from transition.

Related to #3209 other redirects from 10.5

**Completion criteria:**
- [ ] RAD FAQs are migrated

**Comment by bmathesonFEC** [comment]
This tickets was broken off of #3209 because the RAD FAQ work was blocking the the other redirects.

**Comment by bmathesonFEC** [comment]
The RAD managers have elected to replace the RAD FAQ link in the RFAIs with the following links:

- Requests for Additional Information: https://www.fec.gov/help-candidates-and-committees/request-additional-information/
- Main Help for Candidates and Committees page: https://www.fec.gov/help-candidates-and-committees/

**Comment by bmathesonFEC** [comment]
[RAD FAQ](https://transition.fec.gov/rad/) should redirect to the main [H4CC page](https://www.fec.gov/help-candidates-and-committees/). 

I'm currently wrapping up my review of the analyst review on the RAD FAQ content. There is a bunch of content in the Party and PAC FAQ that is not present on the new site.

**Comment by bmathesonFEC** [comment]
Question about redirects for @patphongs, @dorothyyeager, or @kathycarothers:

Do we need to set up a redirect for url with an # anchor within a page? For example:

This page has many anchored links on the page: https://transition.fec.gov/rad/candidates/FEC-ReportsAnalysisDivision-CandidateCommittees.shtml

- https://transition.fec.gov/rad/candidates/FEC-ReportsAnalysisDivision-CandidateCommittees.shtml#GeneralReporting
- https://transition.fec.gov/rad/candidates/FEC-ReportsAnalysisDivision-CandidateCommittees.shtml#NewCandidate
- https://transition.fec.gov/rad/candidates/FEC-ReportsAnalysisDivision-CandidateCommittees.shtml#submitreports

**Comment by dorothyyeager** [comment]
Hi @bmathesonFEC, I took a look at the code for the redirects and didn't see anchors in there, so I think one redirect for the main URL will work OK but let's doublecheck with @patphongs to be sure.

(p.s. The other way around (redirecting to an anchor link) doesn't work.)

**Comment by bmathesonFEC** [comment]
Thanks for checking! Also, good to know about redirecting TO an anchor.

**Comment by JonellaCulmer** [comment]
@bmathesonFEC Should this ticket be closed in favor of this new ticket: https://github.com/fecgov/fec-cms/issues/3924

**Comment by kathycarothers** [comment]
Closing for a fresh ticket after #3924 is completed. The process for redirects has changed since this ticket was first submitted.
